recipe/bld_pytorch.bat

```diff
@@ -42,9 +39,26 @@ set CUDNN_INCLUDE_DIR=%LIBRARY_PREFIX%\include
 set CMAKE_GENERATOR=Ninja
 set "CMAKE_GENERATOR_PLATFORM="
 set "CMAKE_PREFIX_PATH=%LIBRARY_PREFIX%"
+set "CMAKE_INCLUDE_PATH=%LIBRARY_INC%"
+set "CMAKE_LIBRARY_PATH=%LIBRARY_LIB%"
 set "libuv_ROOT=%LIBRARY_PREFIX%"
-set "USE_SYSTEM_SLEEF=OFF"
-set "BUILD_CUSTOM_PROTOBUF=OFF"
+set "USE_SYSTEM_SLEEF=ON"
+set "INSTALL_TEST=0"
+set "BUILD_TEST=0"
+set "MAX_JOBS=%CPU_COUNT%"
+
+@REM There are link errors because of conflicting symbols with caffe2_protos.lib
+set "BUILD_CUSTOM_PROTOBUF=ON"
```

Comment:
This seems suspect. I think we will want to use the system protobuf? If not, could you remove it as a dependency for windows.

Reply:
Okay, I can mark protobuf with `[unix]`.

Reply:
Maybe we want, but we can't because Caffe2&PyTorch build rules depend heavily on protobuf compiled from third_parties/ folder and not provided externally.